src/ast/errors.ts

```diff
@@ -0,0 +1,37 @@
+type filename = string;
+type lineno = number;
+type offset = number;
+type text = string;
+
+export class pyExc extends Error {
+    static _name = "Exception";
+    args: [string, ...any[]];
+    constructor(...args: [string, ...any[]]) {
+        super(args[0]);
+        this.args = args;
+    }
+    get [Symbol.toStringTag]() {
+        return (this.constructor as typeof pyExc)._name;
+    }
```

Comment:
again similar approach here maybe for converting to a python exception - could do a test in `ExternalError`

Reply:
Or maybe we can make these error types subclass the JavaScript SyntaxError. And catch them that way.